Fabric.Authorization.Domain/Services/EDWAdminRoleSyncService.cs

```diff
@@ -66,14 +78,14 @@ private async Task<bool> IsUserASuperAdmin(User user)
             // if a group is a special Super Admin Group then return true
             // if not, then look on each group to see if there is a "super admin role"
             // finally, look on each group's children groups to see if there is a "super admin role"
-            foreach (var customGroup in user.Groups)
+            foreach (var customGroup in user.Groups.Where(p=> !p.IsDeleted))
             {
                 if (customGroup.Roles.Any(r => RoleManagerConstants.AdminRoleNames.Contains(r.Name)))
                 {
                     return true;
                 }
 
-                if (customGroup.Children.SelectMany(children => children.Roles)
+                if (customGroup.Children.Where(group => !group.IsDeleted).SelectMany(children => children.Roles)
```

Comment:
Same question as above - I'm fairly certain this shouldn't be needed, but if it is, the query used to get the group should be adjusted.